pkg/aws/client.go

```diff
@@ -956,6 +958,37 @@ func (c *awsClient) CreateS3Bucket(bucketName string, region string) error {
 	return nil
 }
 
+func (c *awsClient) DeleteS3Bucket(bucketName string) error {
+	c.emptyS3Bucket(bucketName)
+	bucketInput := &s3.DeleteBucketInput{
+		Bucket: aws.String(bucketName),
+	}
+	_, err := c.s3Client.DeleteBucket(bucketInput)
```

Comment:
Is the `emptyS3Bucket` operation a quick one? Do we need to wait in case the bucket has large items in it that take some time to empty? Or is that not a worry here due to what the bucket *should* contain?

Reply:
It should only contain two small files, if it has anything else the client used it wrongfully as far as we need to know